scan/package.json

```diff
@@ -6,7 +6,7 @@
     "parcel-bundler": "^1.12.4"
   },
   "scripts": {
-    "build": "bsb -make-world && parcel build index.html",
+    "build": "rm -rf dist/ && bsb -make-world && parcel build index.html --no-source-maps --no-cache",
```

Comment:
Why is this change necessary?

Reply:
--no-cache is necessary because sometime it will use the env from cache. So we just remove the cache.
"rm -rf dist/" and "--no-source-maps" are irrelevant to this pr, I just add them to reduce redundant build in dist folder and reduce the size of the bundle.

Reply:
Sounds good!